gradle/check-licenses.gradle

```diff
@@ -46,7 +46,8 @@ ext.acceptedLicenses = [
         'CC0 1.0 Universal License',
         'Common Development and Distribution License 1.0',
         'Unicode/ICU License',
-        'CC0'
+        'CC0',
+        'IAIK of Graz University of Technology License'
```

Comment:
need to investigate if we're happy with Ethsigner using this license - if not, maybe we can look at creating "plugins" for signers - such that this can be dynamically loaded, and its not longer an issue for Ethsigner.